cmd/eip_create.go

```diff
@@ -18,17 +18,55 @@ var eipCreateCmd = &cobra.Command{
 		if len(args) >= 1 {
 			zone = args[0]
 		}
-		return associateIPAddress(zone)
+
+		interval, err := cmd.Flags().GetInt64("healthcheck-interval")
+		if err != nil {
+			return err
+		}
+		mode, err := cmd.Flags().GetString("healthcheck-mode")
+		if err != nil {
+			return err
+		}
+		path, err := cmd.Flags().GetString("healthcheck-path")
+		if err != nil {
+			return err
+		}
+		port, err := cmd.Flags().GetInt64("healthcheck-port")
+		if err != nil {
+			return err
+		}
+		strikesFail, err := cmd.Flags().GetInt64("healthcheck-strikes-fail")
+		if err != nil {
+			return err
+		}
+		strikesOK, err := cmd.Flags().GetInt64("healthcheck-strikes-ok")
+		if err != nil {
+			return err
+		}
+		timeout, err := cmd.Flags().GetInt64("healthcheck-timeout")
+		if err != nil {
+			return err
+		}
+
+		return associateIPAddress(interval, port, strikesFail, strikesOK, timeout, mode, path, zone)
```

Comment:
What about to give an `egoscale.AssociateIPAddress` in param and fill zone id after.
better visibility and less parameters